functions.ts

```
@@ -332,156 +394,17 @@
+    logging.setLoggingLevel(4);
   }
+  logging.setLoggingLevel(5);
 }
 
 export function deactivateDebugMode() {
-  if ("eruda" in window) {
-    if (window.eruda.hasOwnProperty("destroy")) {
-      window.eruda.destroy();
-    }
+  if (typeof window !== "undefined" && "eruda" in window && window.eruda.hasOwnProperty("destroy")) {
+    window.eruda.destroy();
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)